proto/mls/message_contents/message.proto

```diff
@@ -10,7 +10,9 @@ option java_package = "org.xmtp.proto.mls.message.contents";
 message WelcomeMessage {
   // Version 1 of the WelcomeMessage format
   message V1 {
-    bytes welcome_message_tls_serialized = 1;
+    uint64 id = 1;
+    uint64 created_ns = 2;
```

Comment:
Not important, but any reason we need both `id` and `created_ns`? The `created_ns` is server-generated and can't be trusted right?

Reply:
We do need some timestamp to show to the user, even if we don't trust it for protocol operation.